spec/models/preserved_copy_spec.rb

```diff
@@ -229,11 +244,26 @@
 
     before { pc.save! }
 
-    it 'returns PreservedCopies that need fixity check, never checked first, then least-recently to most-recently' do
-      expect(described_class.fixity_check_expired.to_a).to eq [pc, old_check_pc1, old_check_pc2]
+    describe '.fixity_check_expired' do
+      it 'returns PreservedCopies that need fixity check, never checked first, then least-recently to most-recently' do
```

Comment:
i think the second part of this test title (about the order, "never checked first, then least-recently to most-recently") can be removed.